sdk/identity/identity/test/manual/README.md

```diff
@@ -28,25 +28,14 @@ You will need to configure an AAD App Registration as follows:
 - Click on `+ Add a platform`, select `Single-page application`, enter `http://localhost:8080` as the redirect URI, then make sure to include implicit grant for "Access tokens" and "ID tokens".
 - Keep in mind that if you belong to an organization, other restrictions based on the organization configurations might prevent you from authenticating. If These steps don't end up being effective, try again on a personal account.
 
-Grant access to this AAD application to your Key Vault by:
+Grant access to this AAD application to your Service Bus by:
 
-- Creating a Key Vault (if you haven't created one).
-- Either in the "Access policies" section of the creation form, or by going to your Key Vault's "Access policies" page, click con `+ Add Access Policy`, select all permissions, then select your AAD application as the "principal", then click "Add", then click "Save" if applicable.
+- Creating a Service Bus namespace (if you haven't created one).
+- Either in the "Access policies" section of the creation form, or by going to the Service Bus namespace's "Access policies" page, click con `+ Add Access Policy`, select all permissions, then select your AAD application as the "principal", then click "Add", then click "Save" if applicable.
+- Then, in your Service Bus namespace, create a queue named `partitioned-queue`, then go into the queue and repeat the "Access policies" process to add a "principal".
```

Comment:
> `then go into the queue and repeat the "Access policies" process to add a "principal".`

Why is this needed? (if you have already added the AAD app to the whole namespace in the previous step)

Reply:
I'll remove it!